src/components/contexts/MapContextProvider.jsx

```diff
@@ -24,8 +24,8 @@ const MapContextProvider = ({children}) => {
   const [tiles, setTiles] = useState([]);
   const [rerenderTiles, setRerenderTiles] = useState(false);
   const [connectedStopIds, setConnectedStopIds] = useState([]);
-  const [connectedStopVisibility, setConnectedStopVisibility] = useState('Visible');
-  const [connectionLineVisbility, setConnectionLineVisbility] = useState('Visible');
+  const [connectedStopVisibility, setConnectedStopVisibility] = useState('Semi-transparent');
```

Comment:
Generalnie okej, ale na przyszłość, jeżeli nie masz TypeScripta, to szansa na literówce w stringu jest wysoka.
Raczej warto trzymać w jakimś pliku ze stałymi `const visible = 'visible'; const semiTransparent = 'Semi-Transparent';`
Wtedy używasz `useState(semiTransparent)` i nie walniesz literówki